composer.json

```diff
@@ -1,30 +1,49 @@
 {
     "name": "outlandish/wpackagist",
     "description": "Install and manage WordPress plugins with Composer",
+    "config": {
+        "sort-packages": true
+    },
+    "minimum-stability": "stable",
     "require": {
-        "php": ">=5.3.2",
-        "composer/composer": "1.3.*",
-        "silex/silex": "~1.1",
-        "twig/twig": ">=1.8,<2.0-dev",
-        "symfony/console": "*",
-        "symfony/filesystem":"*",
-        "symfony/twig-bridge": "~2.3",
-        "symfony/form": "~2.3",
-        "symfony/security-csrf": "~2.3",
-        "symfony/locale": "~2.3",
-        "symfony/config": "~2.3",
-        "symfony/translation": "~2.3",
-        "pagerfanta/pagerfanta": "dev-master",
-        "franmomu/silex-pagerfanta-provider": "dev-master",
-        "doctrine/dbal": "2.5.*",
-        "knplabs/console-service-provider": "1.0",
-        "rarst/wporg-client": "dev-master",
-        "guzzlehttp/guzzle-services": "1.0.*"
+        "php": ">=7.2",
```

Comment:
wpackagist.org only has php@7.1.17, so would need to either upgrade php or downgrade this requirement

Reply:
@rasmuswinter since the 7.1 branch is now out of even security patch support I would say upgrading is the right move.

Reply:
In that case, I'm afraid I can't code review or test any further until I upgrade my local php - I only have php v7.2.0, and this requires 7.2.5